src/ameba/runner.cr

```diff
@@ -159,6 +160,10 @@ module Ameba
 
       File.write(source.path, source.code) unless corrected_issues.empty?
     ensure
+      missing_location = Crystal::Location.new(nil, 0, 0)
+      source.issues.sort_by! do |issue|
+        issue.location || missing_location
+      end
```

Comment:
didn't test, but how will this work in parallel env?

Reply:
Should be fine, since `run_source` is being run per fiber:

https://github.com/crystal-ameba/ameba/blob/29989b06c8fb5543d18221914a188cf684d5774c/src/ameba/runner.cr#L106-L114